test/csuite/wt3184_dup_index_collator/main.c

```diff
@@ -62,8 +70,7 @@ static void
 print_int_item(const char *str, const WT_ITEM *item)
 {
     if (item->size > 0) {
```

Comment:
Formatting: If you're removing the assert, then it's only one line and you can remove these braces.

Reply:
Done, thanks!